CHANGELOG.md

```diff
@@ -23,8 +23,10 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 * ### ALL
     * #### Added
+        * Support for Python 3.8
```

Comment:
Didn't we do this earlier? I suppose now it's "official"?

Reply:
Yes, until the issue with scipy was fixed (it is now) we didn't say we officially supported 3.8